Sources/ArcGISToolkit/Components/Offline/OfflineManager.swift

```diff
@@ -17,7 +17,49 @@ import Combine
 import OSLog
 import SwiftUI
 
-/// An object that maintains state for the offline components.
+/// A utility class that maintains states of offline map areas and their
+/// file structures on the device.
+///
+/// This component is based on ``JobManager`` and provides high-level APIs to
+/// manage offline map areas and access their data. A custom UI can be built
+/// using the APIs provided.
+///
+/// **Features**
+///
+/// The component supports both ahead-of-time(preplanned) and on-demand
+/// workflows for offline mapping. It allows you to:
+///
+/// - Observe job status.
+/// - Manage offline map info so the offline map areas can be loaded while the
+/// device is disconnected.
+/// - Remove offline map areas from the device.
+/// - Run the jobs while the app is in background or even terminated.
+/// - Get notified when the jobs complete.
+///
+/// **Behavior**
+///
+/// The offline manager is not instantiable, you must use the ``shared`` instance.
+/// You can also use the `offlineManager(preferredBackgroundStatusCheckSchedule:jobCompletion:)`
+/// modifier to set up the offline manager for the app. For example:
```

Comment:
```suggestion
/// You must set the `offlineManager(preferredBackgroundStatusCheckSchedule:)` modifier at the entry point of your 
/// application to add additional setup required for the component to use the job manager. For example:
```
This view modifier is required so that download jobs run in the app background.